travis/tests/secondorder/model/configuration/secondorder.fac

```diff
@@ -0,0 +1,27 @@
+********************************************************************* ;
+* This mechanism models 2 species with a single reaction converting * ;
+* one to the other with a doubled rate. In the general case with    * ;
+* rate k, the solution is                                           * ;
+* O(t) = 1/(1/O(0) + kt)                                            * ;
+* O3(t) = O3(0) + O(0) - 1/(1/O(0) + kt)                            * ;
+********************************************************************* ;
+*;
+* Generic Rate Coefficients ;
+*;
+* Complex reactions ;
+*;
+****************************************************** ;
+*;
+* Peroxy radicals. ;
+*;
+* WARNING: The following species do not have SMILES strings in the database. ;
+*          If any of these are peroxy radicals the RO2 sum will be wrong!!! ;
+****************************************************** ;
+*  ;
+RO2 = ;
+*;
+* Reaction definitions. ;
+*;
+% 0.0001*(TEMP/300) : O + O = O3 + O3 ;
```

Comment:
Just to be clear. This case basically also checks that `A+A=B` and `A+A=B+C` are correct, right?

Reply:
While it doesn't explicitly do so, this case is probably "more complex" than either of those, so it provides some reassurance that they would behave as expected too.

Reply:
That's what I thought. Very good. :)